roles/istio/defaults/main.yaml

```diff
@@ -8,5 +8,6 @@ istio_create_seldon_gateway: true
 istio_use_ssl: false
 istio_self_signed_certs: "{{ istio_use_ssl }}"
 istio_ingress_cert_secret: "seldon-deploy-cert-tls"
+istio_external_ip: ""
```

Comment:
Doesn't `istio_external_ip` exist only after execution of this `role`? And for the certificates this role sets this variable for its own purposes